### PR TITLE
Feat/#42 landing page

### PR DESCRIPTION
**Pull request by N4choWasTaken**
Deleted sus button
Created view for Laptop and Smartphone

Closes Oh-my-class/oh-my-class#42

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -2,7 +2,14 @@ import type { NextPage } from "next";
 import React from "react";
 
 const Home: NextPage = () => {
-  return <button className="btn">Kinda sus</button>;
+  return <div data-theme="winter" className=" w-screen h-screen absolute flex flex-row justify-center bg-base-100 font-sans bg-gradient-to-t from-white via-white to-slate-600"> 
+    <div className=" w-9/12 text-center mt-14 md:mt-8">
+      <img src="https://www.theanimator.co.in/icon/1998840123schoolmanagement-icon.png" alt="School Icon" className="md:w-1/4 md:h-1/8 mx-auto"/>
```
Comment (Lezurex) [inline pages/index.tsx:7]: Do we have the copyright for this image? Would be better to serve it directly from ourselves.

**Review by Lezurex — CHANGES_REQUESTED** [review]

File: pages/index.tsx [hunk 2]
```diff
@@ -2,7 +2,14 @@ import type { NextPage } from "next";
...
+      <h1 className=" text-slate-800 font-bold text-5xl">Oh my Class</h1>
+      <p className=" text-slate-600 text-xl font-bold mt-7">Make your school life at least better than suicidal thoughts!</p>
```
Comment (Z-100) [inline pages/index.tsx:9]: Is this family friendly?

File: pages/index.tsx [hunk 3]
```diff
@@ -2,7 +2,14 @@ import type { NextPage } from "next";
 import React from "react";
 
 const Home: NextPage = () => {
-  return <button className="btn">Kinda sus</button>;
+  return <div data-theme="winter" className=" w-screen h-screen absolute flex flex-row justify-center bg-base-100 font-sans bg-gradient-to-t from-white via-white to-slate-600"> 
+    <div className=" w-9/12 text-center mt-14 md:mt-8">
+      <img src="https://www.theanimator.co.in/icon/1998840123schoolmanagement-icon.png" alt="School Icon" className="md:w-1/4 md:h-1/8 mx-auto"/>
+      <h1 className=" text-slate-800 font-bold text-5xl">Oh my Class</h1>
```
Comment (Z-100) [inline pages/index.tsx:8]: Capitalize each word, as "Oh My Class" sounds more interesting and clickbatey.

File: pages/index.tsx [hunk 4]
```diff
@@ -3,10 +3,10 @@ import React from "react";
...
 const Home: NextPage = () => {
   return <div data-theme="winter" className=" w-screen h-screen absolute flex flex-row justify-center bg-base-100 font-sans bg-gradient-to-t from-white via-white to-slate-600"> 
-    <div className=" w-9/12 text-center mt-14 md:mt-8">
-      <img src="https://www.theanimator.co.in/icon/1998840123schoolmanagement-icon.png" alt="School Icon" className="md:w-1/4 md:h-1/8 mx-auto"/>
-      <h1 className=" text-slate-800 font-bold text-5xl">Oh my Class</h1>
-      <p className=" text-slate-600 text-xl font-bold mt-7">Make your school life at least better than suicidal thoughts!</p>
+    <div className=" w-9/12 text-center mt-20 md:mt-8">
+      <img src="https://cdn.onlinewebfonts.com/svg/img_142809.png" alt="School Icon" className="md:w-1/4 md:h-1/8 mx-auto"/>
+      <h1 className=" text-slate-800 font-bold text-5xl mt-12">Oh my Class</h1>
+      <p className=" text-slate-600 text-xl font-bold mt-7">Make your school life better with Oh my Class!</p>
```
Comment (Z-100) [inline pages/index.tsx:9]: Capitalize each word of "Oh My Class", as it's a name.

**Review by Lezurex — APPROVED** [review]

**Pull request merged**